lib/decision_review_v1/utilities/form_4142_processor.rb

```diff
@@ -66,7 +67,12 @@ def generate_metadata
       end
 
       def received_date
```

Comment:
when does this run?  If it's running on a worker, then this would not necessarily fix the problem 🤔 since some workers can take a day to run, e.g. submission at 9pm might succeed on it's 13th retry the next day.  We need to be sure that the moment the vet clicks `submit` on 526, that date is signed on this (and all other) ancillary forms.

Reply:
Other workflows use this processor, so for our use case, I'm trying to match whatever happens during the normal 526 submission workflow, which is setting it on the saved_claim's created_at date. I don't want to mess up the timestamping for other teams since they haven't noticed any issues.

Reply:
I think we can just safely revmove the `Time.now...` case and that will prevent accidentally adding the wrong date

Reply:
@tommasina-va @SamStuckey Is the `Time.now` on line 73 okay?

Reply:
@rmtolmach yeah, that's for other workflows using this 4142 processor. Sam approved in this [comment](https://github.com/department-of-veterans-affairs/vets-api/pull/14900#issuecomment-1863545102). Thanks for checking!

Reply:
ah, I missed that. Thanks!